index.js

```diff
@@ -24,7 +24,9 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
 
-    app.import('vendor/axe-core/axe.js', { type: 'test' });
+    if (app.env !== 'production') {
```

Comment:
Should be if `app.tests` (which handles the `ember test --environment=production` case also)